source/_integrations/influxdb.markdown

```diff
@@ -37,41 +39,64 @@ You will still need to create a database named `home_assistant` via InfluxDB's c
 {% configuration %}
 host:
   type: string
-  description: IP address of your database host, e.g., 192.168.1.10
+  description: "**1.xx only** IP address of your database host, e.g., 192.168.1.10"
   required: false
   default: localhost
 port:
   type: integer
-  description: Port to use
+  description: "**1.xx only** Port to use"
   required: false
   default: 8086
 username:
   type: string
-  description: The username of the database user. The user needs read/write privileges on the database
-  required: false
+  description: "**1.xx only** The username of the database user. The user needs read/write privileges on the database."
+  required: inclusive
 password:
   type: string
-  description: The password for the database user account.
-  required: false
+  description: "**1.xx only** The password for the database user account. Needed with `username` configuration variable."
+  required: inclusive
 database:
   type: string
-  description: Name of the database to use. The database must already exist.
+  description: "**1.xx only** Name of the database to use. The database must already exist."
   required: false
   default: home_assistant
 ssl:
   type: boolean
-  description: Use HTTPS instead of HTTP to connect.
+  description: "**1.xx only** Use HTTPS instead of HTTP to connect."
   required: false
   default: false
 verify_ssl:
   type: boolean
-  description: Verify SSL certificate for HTTPS request.
+  description: "**1.xx only** Verify SSL certificate for HTTPS request."
   required: false
   default: true
 path:
   type: string
-  description: Path to use if your InfuxDB is running behind an reverse proxy.
+  description: "**1.xx only** Path to use if your InfuxDB is running behind an reverse proxy."
   required: false
+api_v2:
+  type: boolean
+  description: "**2.xx only** Set to `true` if connecting to a 2.xx installation, do not use otherwise."
+  required: inclusive
+  default: false
+url:
+  type: string
+  description: "**2.xx only** URL of your 2.xx installation (e.g. http://localhost:9999)."
+  required: false
+  default: "https://us-west-2-1.aws.cloud2.influxdata.com"
```

Comment:
Can't this be merged with the host/port configuration of v1?
Basically, even v1 API is an URL endpoint

Reply:
Yep you're right, modified the two PRs accordingly.  `url` option is gone, building the URL from the other config options.